### DISCUSSION TITLE
Import keys from ssh agent

### DISCUSSION THREAD

**Discussion by ghost** [devlinx9/muon-ssh#57] [Ideas]
Hello, i am using  your ssh client and its awesome. 
But i have many sessions for login via private key file and its annoying write password for every private key if i have it load in ssh agent.
I think, that many users will approciate that. Do u plan add this feature ? 
Regards 
Jan

**Comment by devlinx9** [comment]
Hello, 

I'm glad that you like muon
For the feature, yes this additional password for private key is something that I want to add in the client.

**Reply by BayJay** [reply]
when I enter the password from the key, he writes that the password is not correct
